src/speculators/models/eagle3/config.py

```
@@ -82,3 +82,94 @@ def validate_transformer_config(cls, value: Any) -> PretrainedConfig:
+    
+    @property
+    def pre_vwn_layer_class(self) -> type:
+        from .model_definitions import VwnEagle3PreVwnLayer, VwnEagle3PreVwnLayerV1
+        """根据 pre_vwn_version 返回对应的 pre_vwn_layer 类。"""
+        if self.pre_vwn_version == 0:
+            return VwnEagle3PreVwnLayer
+        elif self.pre_vwn_version == 1:
+            return VwnEagle3PreVwnLayerV1
+        else:
+            raise ValueError(
+                f"未知的 pre_vwn_version: {self.pre_vwn_version}，"
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)